src/Adapters/LinuxSystemResources.php

```diff
@@ -68,4 +68,19 @@ public function ramUsedResourcesPercentage()
     {
         return $this->ramResources() / $this->ramTotalResources() * 100;
     }
+
+    /**
+     * Retrieve the disk usage.
+     *
+     * @return int
+     */
+    public function diskUsedResourcesPercentage()
+    {
+        $totalDiskSpace = disk_total_space('/');
```

Comment:
The variables should be written with snake case. You are following the PSR-2 standard which would be correct since this is a Laravel project, but this project uses snake case for its variables. This applies for all variables.